turbolinks/src/main/java/com/basecamp/turbolinks/TurbolinksView.java

```diff
@@ -122,23 +124,36 @@ void hideProgress() {
      * <p>Attach the shared webView to the TurbolinksView.</p>
      *
      * @param webView The shared webView.
+     * @param swipeRefreshLayout parent view of webView
      * @param screenshotsEnabled Indicates whether screenshots are enabled for the current session.
      */
-    void attachWebView(WebView webView, boolean screenshotsEnabled) {
-        if (webView.getParent() == this) return;
+    void attachWebView(WebView webView, TurbolinksSwipeRefreshLayout swipeRefreshLayout, boolean screenshotsEnabled, boolean pullToRefreshEnabled) {
+        if (swipeRefreshLayout.getParent() == this) return;
 
-        if (webView.getParent() instanceof TurbolinksView) {
-            TurbolinksView parent = (TurbolinksView) webView.getParent();
+        if (!pullToRefreshEnabled) swipeRefreshLayout.setEnabled(false);
```

Comment:
You don't need the `if` conditional here if you just use in the passed in `pullToRefreshEnabled`. 😉